hack/postrelease/golang/README.md

```diff
@@ -0,0 +1,32 @@
+# Open-source postrelease tests!
+
+## How to run
+
+You can use ginkgo to run the tests:
+
+```
+$ ginkgo run tests/oss
```

Comment:
Thought we wanted to move away from `ginkgo`.

Reply:
oops, will correct

Reply:
Fixed!